pandas/core/window/rolling.py

```diff
@@ -299,6 +299,7 @@ def __iter__(self):
             center=self.center,
             closed=self.closed,
         )
+
```

Comment:
Could you undo these whitespaces (and the rest of them in this PR)?

Reply:
Hmmph, I thought I had.  Not sure what's going on, but I'll keep looking at the diff until I don't see them any more.